utils/docker/Dockerfile.centos.7

```diff
@@ -1,198 +0,0 @@
-# Copyright 2018-2022 Intel Corporation
```

Comment:
You can't remove this file without removing https://github.com/daos-stack/daos/blob/cf420e9786d2168d096fc63a4ae786754ea25263/Jenkinsfile#L420-L484

Reply:
ah, right, but i cannot remove this entry yet due to NLT & UT. Let me re-add it for now.

Reply:
Maybe it's a GitHub review artifact, but it looks like this is still removed and there are still references to it in the `Jenkinsfile`.